bin/re_drive_prod

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+read -p "What is the ID of the SNAP Application you want to re-drive? "
+SNAP_APPLICATION_ID=$REPLY
+echo
+
+read -p "What is the name of the Heroku remote you'd like to access? (staging/production) "
+echo
+
+HEROKU_REMOTE=$REPLY
+
+read -p "WARNING: you are about to access the $HEROKU_REMOTE database directly.
+Are you sure you want to proceed? (y/n) " -n 1 -r
+echo
+
+if [[ $REPLY =~ ^[Yy]$ ]];then
+  echo "OK, proceeding with accessing $HEROKU_REMOTE database..."
+else
+  echo "OK, bye bye!"
+  exit 1
+fi
+echo
+
+read -p "Have you set up the development environment to use DATABASE_URL in the
+database.yml file? (y/n) " -n 1 -r
```

Comment:
Do we need to do this if we're running the rails app in the production env?

Reply:
No but we are not running this in the production env anymore because I ran
into too many issues going that route